src/app/test_executive/payments_test.ml

```diff
@@ -36,13 +35,13 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     { default with
       requires_graphql = true
     ; block_producers =
-        [ { balance = "40000"; timing = Untimed }
-        ; { balance = "30000"; timing = Untimed }
+        [ { balance = "40000"; timing = Untimed } (* 40_000_000_000_000 *)
+        ; { balance = "30000"; timing = Untimed } (* 30_000_000_000_000 *)
         ; { balance = "10000"
           ; timing =
-              make_timing ~min_balance:1_000_000_000_000 ~cliff_time:8
+              make_timing ~min_balance:10_000_000_000_000 ~cliff_time:8
```

Comment:
Is this ok? I believe the timing test (which is currently failing) depends on these values

Reply:
ah i am tinkering with this at the moment, i'll get it passing.  as long as it's passing you don't mind if i jack up the values do you?  I'm just trying to increase the values to that coinbases stop messing with my math.

Reply:
Ah ok, that's fine. You could use the new extra genesis accounts field in test config so that you don't have to deal with coinbase amounts

Reply:
oh that's a good idea, maybe in a different PR tho